pynest/nest/tests/test_connect_arrays.py

```diff
@@ -51,19 +74,29 @@ def test_connect_arrays(self):
             self.assertEqual(c.weight, w)
             self.assertEqual(c.delay, d)
 
+    def test_connect_arrays_no_conn_spec(self):
+        """Connecting NumPy arrays of node IDs"""
+        n = 10
+        nest.Create('iaf_psc_alpha', n)
+        sources = np.arange(1, n+1, dtype=np.uint64)
+        targets = non_unique
+
+        with self.assertRaises(ValueError):
+            nest.Connect(sources, targets)
+
     def test_connect_arrays_threaded(self):
         """Connecting NumPy arrays, threaded"""
         nest.SetKernelStatus({'local_num_threads': 2})
         n = 10
         nest.Create('iaf_psc_alpha', n)
         sources = np.arange(1, n+1, dtype=np.uint64)
-        targets = np.arange(1, n+1, dtype=np.uint64)
+        targets = non_unique
         weights = np.ones(len(sources))
         delays = np.ones(len(sources))
         syn_model = 'static_synapse'
 
-        nest.Connect(sources, targets, syn_spec={'weight': weights, 'delay': delays,
-                                                 'synapse_model': syn_model})
+        nest.Connect(sources, targets, conn_spec="one_to_one",
+                     syn_spec={'weight': weights, 'delay': delays, 'synapse_model': syn_model})
```

Comment:
Use `'` or `"` for strings consistently throughout the file (docstrings are always with `"`s however).